src/client.ts

```diff
@@ -222,6 +222,19 @@ export default class Client {
     const env = process.env;
     const useYjit = vscode.workspace.getConfiguration("rubyLsp").get("yjit");
 
+    [
+      "RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR",
+      "RUBY_GC_HEAP_FREE_SLOTS",
+      "RUBY_GC_HEAP_SLOTS_GROWTH_FACTOR",
+      "RUBY_GLOBAL_METHOD_CACHE_SIZE",
+      "RUBY_GC_MALLOC_LIMIT",
+      "RUBY_GC_HEAP_GROWTH_MAX_SLOTS",
+      "RUBY_GC_OLDMALLOC_LIMIT",
+      "RUBY_GC_HEAP_INIT_SLOTS",
```

Comment:
Other than looking at the Ruby source, is there somewhere that documents all the available variables? Google isn't returning anything useful.

Reply:
The most sensible thing might be to remove all the env var that start with `RUBY_GC_*`, since we introduced new ones in Ruby 3.3.0-dev already, it would be more future proof.

Reply:
Makes sense: https://github.com/Shopify/vscode-ruby-lsp/issues/388

Reply:
https://github.com/Shopify/vscode-ruby-lsp/pull/389